### PR TITLE
Dev/ray/linkfixes

### PR DESCRIPTION
**Pull request by rayreed137**
## Description

Added development guide to ".github/" folder and fixed linked in readme

This repository needs 

The commits in this pull request will 

## Changes

- Imperative commit or change title (commit ID) 
  - Add some bullet points to explain the change
- Next commit or change (commit ID)

## Related

Relates to organization/repo#number

Closes organization/repo#number

- [ ] I have reviewed the [Guidelines for Contributing](CONTRIBUTING.md) and the [Code of Conduct](CODE_OF_CONDUCT.md).

### CODE REVIEW

File: .github/DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+```bash
+# Build everything, producing a source tar and wheel in ./dist
+make
+
+# For running tests locally
+make test
+
+# For running notebook tests
+make test-notebooks
```
Comment (FelipeAdachi) [inline .github/DEVELOPMENT.md:55]: we still don't have that command, but is TODO. maybe we should remove this mention until we have it?

Comment (jamie256) [reply .github/DEVELOPMENT.md:55]: same the bump version targets, e.g. we need to bring back `bump-minor`

**Review by jamie256 — CHANGES_REQUESTED** [review]
Can you run from the python folder:
`make pre-commit`
Then add the updates (if any to your changed files) to the branch? I think the whylogs-ci is failing on some whitespace formatting on the README.md

File: .github/DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+Poetry manages virtualenvs as well. Typically, on a project that uses virtualenv directly you would activate the virtualenv to get all of the binaries that you install with pip onto the path. Poetry works in a similar way but with different commands.
+
+```
+# Activate the poetry virtualenv
+poetry shell
+```
+
+You shouldn't have to actually do that unless you want to run some of the executables that are installed from pypi directly. One use case would be manually running `pytest` on a single test file rather than running them all with `make test`.
+
+## Manging Dependencies
```
Comment (jamie256) [inline .github/DEVELOPMENT.md:30]: Typo: Manging -> Managing

**Comment by TheMellyBee** [comment]
CU-2m9ez49

**Comment by andyndang** [comment]
Task linked: [CU-2m9ez49 Update links PR](https://app.clickup.com/t/2m9ez49)

**Comment by TheMellyBee** [comment]
I agreed to take this, but don't want to work on Ray's fork of whylogs. This is the current PR.  #698 Directly pulled from this with the changes requested made

**Pull request closed without merging**